lib/engine/step/g_1846/train.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+require_relative '../train'
+
+module Engine
+  module Step
+    module G1846
+      class Train < Train
+        def setup
+          super
+          @issued = false
+        end
+
+        def actions(entity)
+          return [] if entity.minor? || entity.company?
```

Comment:
unless entity.corporation?

Reply:
player is still allowed due to EMR

324c7fae